metro.config.js

```diff
@@ -1,4 +1,37 @@
 // Learn more https://docs.expo.io/guides/customizing-metro
 const { getDefaultConfig } = require('expo/metro-config');
 
-module.exports = getDefaultConfig(__dirname);
+module.exports = (async () => {
+  const {
+    resolver: { assetExts }
+  } = await getDefaultConfig(__dirname);
+
+  return {
+    transformer: {
+      getTransformOptions: async () => ({
+        transform: {
+          experimentalImportSupport: false,
+          inlineRequires: true
+        }
+      })
+    },
+    resolver: {
+      assetExts: [
+        ...assetExts,
+        'obj',
+        'mtl',
+        'fbx',
+        'mp3',
+        'JPG',
+        'vrx',
+        'fbx',
+        'hdr',
+        'gltf',
+        'glb',
+        'bin',
+        'arobject',
+        'gif'
```

Comment:
do we need all of these extensions?

Reply:
We don't know exactly what the real data will be. I wanted to add all the variations, I can update this according to the data coming in. I also needed this to experiment with different file types.

Reply:
i understand, but if we say we need fbx and mp3, maybe obj and mtl, images, then we should be able to remove these ones:

```
'hdr',
'gltf',
'glb',
'bin',
'arobject',
'gif'
```

what if the trigger image will be jpeg or png? maybe we need to add the two options?
and why is `JPG` the only one in capital letters. does it have some effect if it would be `jpg`? can you ask the viro community about this?

Reply:
I prepared the link to the document in advance because I knew you would ask this question. I learnt that when adding an asset, it must be written in a case sensitive way. Therefore, JPG must be written.

[Viro Document](https://viro-community.readme.io/docs/assets#adding-asset-types-to-react-native)

Reply:
ok, so we can remove the ones we do not need.
is there a way to log `assetExts` to see whtas included to think bout adding other image types in specific case?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/568/commits/793b9476e9084bf158662592745d73144792a8a0

Reply:
I have encountered different file types that can be used.

```
[
  'bmp',  'gif',  'jpg', 'jpeg',
  'png',  'psd',  'svg', 'webp',
  'm4v',  'mov',  'mp4', 'mpeg',
  'mpg',  'webm', 'aac', 'aiff',
  'caf',  'm4a',  'mp3', 'wav',
  'html', 'json', 'pdf', 'yaml',
  'yml',  'otf',  'ttf', 'zip'
]

Reply:
is this the log output? so we do not need to add `mp3` on our own. but maybe we need `JPEG`, `PNG` like `JPG`, right?

Reply:
Yes, that's right. I think we only need the types `'mtl', 'obj', 'vrx'` for now.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/568/commits/c18f926c1016ad2c779fd9a9338ba5eca0af2c2c